<launch>
   <node name="image_converter"   pkg="magno_test"   type="ion3_twyg_control.py" required="true"/>
   <node name="shuffle_sun"   pkg="basic_led_strip_ros"   type="shuffle_sun_node.py" required="true" />
   <!-- <include file = "" /> -->
</launch>
